ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.hbs

```diff
@@ -28,82 +28,101 @@
+      {{#if (get @model.hostSet 'isStatic')}}
+        <Hds::Table
+          @valign='middle'
+          @model={{@model.hosts}}
+          @columns={{array
+            (hash label=(t 'form.name.label'))
+            (hash label=(t 'form.type.label'))
+            (hash label=(t 'form.id.label'))
+            (hash label=(t 'titles.actions'))
+          }}
+        >
+          <:body as |B|>
+            <B.Tr>
+              <B.Td>
+                <LinkTo
+                  @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
+                  @model={{B.data.id}}
                 >
-                  <code>{{host.id}}</code>
-                </Copyable>
-              </row.cell>
-              <row.cell>
+                  {{B.data.displayName}}
+                </LinkTo>
+                <Hds::Text::Body @tag='p'>
+                  {{B.data.description}}
+                </Hds::Text::Body>
+              </B.Td>
+              <B.Td>
+                <HostCatalogTypeBadge @model={{B.data}} />
+              </B.Td>
+              <B.Td>
+                <Hds::Copy::Snippet
+                  @textToCopy={{B.data.id}}
+                  @color='secondary'
+                />
+              </B.Td>
+
+              <B.Td>
                 {{#if (can 'removeHosts hostSet' @model.hostSet)}}
-                  <Rose::Dropdown
-                    @icon='flight-icons/svg/more-horizontal-16'
-                    @iconOnly={{true}}
-                    @showCaret={{false}}
-                    @dropdownRight={{true}}
-                    @text={{t 'actions.manage'}}
-                    as |dropdown|
-                  >
-                    <dropdown.button
-                      @style='danger'
+                  <Hds::Dropdown as |dd|>
+                    <dd.ToggleIcon
+                      data-test-host-set-hosts-dropdown-toggle
+                      @text={{t 'actions.manage'}}
+                      @icon='more-horizontal'
+                      @hasChevron={{false}}
+                    />
+                    <dd.Interactive
+                      data-test-host-set-hosts-dropdown-remove-host
+                      @text={{t 'actions.remove'}}
+                      @color='critical'
                       {{on
                         'click'
-                        (route-action 'removeHost' @model.hostSet host)
+                        (route-action 'removeHost' @model.hostSet B.data)
                       }}
-                    >
-                      {{t 'actions.remove'}}
-                    </dropdown.button>
-                  </Rose::Dropdown>
+                    />
+                  </Hds::Dropdown>
                 {{/if}}
-              </row.cell>
-            </body.row>
-          {{/each}}
-        </table.body>
-      </Rose::Table>
+              </B.Td>
+
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+      {{else}}
+        <Hds::Table
+          @valign='middle'
```

Comment:
None blocking: Perhaps nit picky, but can we put the `@valign` attribute the last one? so `@model` and `@columns` are first. This is how we have been defining HDS table attributes in the previous work and personally, I think it is good practice to have more important attributes first. Totally debatable and open to it.

Reply:
@calcaide absolutely agreed